app/src/main/java/com/door43/translationstudio/core/TargetTranslation.java

```diff
@@ -185,6 +189,27 @@ public String getTargetLanguageId() {
     }
 
     /**
+     * get format of translation
+     * @return
+     */
+    public TranslationFormat getFormat() {
+        return mTranslationFormat;
+    }
+
+    private TranslationFormat readTranslationFormat() {
+        String formatStr = manifest.getString("format");
+        TranslationFormat format = TranslationFormat.get(formatStr);
+        if(null == format) {
```

Comment:
@neutrinog - in testing, I think we need to make the change that if type is text and project id is not obs, then the format returned is usx.  In this case it will force an upgrade to usfm.